### PR TITLE
Refactor: Es6 and modules

### PR DESCRIPTION
**Pull request by Wineshuga**
**Description**

In this branch, I carried out the following: 

- I created a package.json file using `npm init -y`.
- I created the entry point for your JavaScript code called index.js.
- I created an index.css file and an index.html file. 
- I linked the index.js and index.css files in the index.html file.
- I created a directory called modules.

I re-created my Awesome Books app following the rules:

1. Used ES6 syntax.
2. Divided your code into modules.
3. Created a separate file for each module in the modules directory.
4. Imported modules into the index.js file.
5. Used the library [Luxon](https://moment.github.io/luxon/#/install?id=es6) to display current date ant time.

### CODE REVIEW

File: modules/date.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function displayDate() {
```
Comment (Zuhaib042) [inline modules/date.js]: - [ ] Kindly consider using arrow function syntax from ES6 as it is a requirement for this milestone.

File: modules/showElement.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const userBooks = document.querySelector('.book-list-sec');
+const formPage = document.querySelector('.form-sec');
+const contactPage = document.querySelector('.contact');
+const listBtn = document.querySelector('.list-btn');
+const addNewBtn = document.querySelector('.add-new-btn');
+const contactBtn = document.querySelector('.contact-btn');
+
+export default function showElement(element) {
```
Comment (Zuhaib042) [inline modules/showElement.js]: - [ ] Kindly consider using arrow function syntax from ES6 as it is a requirement for this milestone.

File: modules/submitForm.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import BookList from './classes.js';
+
+const bookList = new BookList();
+
+export default function submitForm(event) {
```
Comment (Zuhaib042) [inline modules/submitForm.js]: - [ ] Kindly consider using arrow function syntax from ES6 as it is a requirement for this milestone.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Wineshuga 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✅ Design is awesome
✅ Good way of using classes and modules

## Required Changes ♻️
- [ ] The license link in your readme is broken. kindly consider adding a `License.md` file in your repo and check if the link is working before submitting again as having broken links in our readme is unprofessional.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @Wineshuga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Clean code
✅ Use of ES6

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**